src/test/java/seedu/address/logic/commands/room/AllocateRoomCommandTest.java

```diff
@@ -28,6 +23,7 @@
 /**
  * Contains integration tests and unit tests for AllocateRoomCommand.
  */
+//TODO: Please clean up test cases for this method, seems weird to have only rooms 7-10 in a hotel facility.
```

Comment:
I think it's fine to have so few rooms in the test cases since we are just testing that the functions work as expected rather than testing if the app works on a large scale. What do you think?

Reply:
yup, i think it should be fine since you are doing unit testing to ensure that the function works